dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporterTest.java

```diff
@@ -233,7 +246,7 @@ public void setUp() {
      * Test of exportDataset method, of class SchemaDotOrgExporter.
      */
     @Test
-    public void testExportDataset() throws Exception {
+    public void testExportDataset_sameLicenses() throws Exception {
```

Comment:
This test method is about much more then licenses. If we want to separate license checking with the rest of json-ld then I would propose to make separate method for the case `_sameLicenses` and in assert in this method just make `assertNotNull(json2.getJsonObject("license"));` instead of full element check